docs/fleet/cli.md

```diff
@@ -363,10 +369,20 @@ viam data database hostname --org-id=abc
 | `export`      | export data in a specified format to a specified location  | - |
 | `database configure`      | create a new database user for the Viam organization's MongoDB Atlas Data Federation instance, or change the password of an existing user. See [Configure data query](/data/query/#configure-data-query)  | - |
 | `database hostname`      | get the MongoDB Atlas Data Federation instance hostname and database name. See [Configure data query](/data/query/#configure-data-query)  | - |
+| `dataset add`      | add a new image to an existing dataset | `filter` |
+| `dataset remove`      | remove an existing image from a dataset | `filter` |
 | `delete binary`      | delete binary data  | - |
 | `delete tabular`      | delete tabular data  | - |
 | `--help`      | return help      | - |
 
+##### Positional arguments: `database`
+
+<!-- prettier-ignore -->
+| argument | description |
+| ----------- | ----------- | ----------- |
+| `filter`     | `add` or `delete` images from a dataset using a filter. See [Using the `filter` parameter)](#using-the-filter-parameter).|
```

Comment:
Also `ids` below to `add` or `delete` images from a dataset using their IDs (rather than a filter)

Reply:
Ah I think I see the confusion: line 372,373 must cover both by-id and using filter. Line 383 is only for usage with a filter. Update initial descriptions of `add` and `remove` to better cover both possible uses. LMK if I am closer!

Reply:
Responded in other comment!